### PR TITLE
Master logical led

### PR DESCRIPTION
**Pull request by davids5**
This PR is the groundwork to fix the power LED blinking on V5

   Background:
   ----------
   Early boards only had an AMBER LED that was used to
   indicate a High Load condition.

   This change defines the new logical interface
   the LED_<color> should not be used in application
   code moving forward, only the manipulator macros
   should be used.

       Logical usage             Legacy default
      ------------------------+-------------
       BOARD_OVERLOAD_LED     | LED_RED

   Later boards defined BOARD_HAS_CONTROL_STATUS_LEDS
   and added the use of a BLUE and GREEN LED that were
   used as follows:

   Logical usage             Legacy default
      ------------------------+-------------
       BOARD_ARMED_LED        | LED_BLUE
       BOARD_ARMED_STATE_LED  | LED_GREEN

   With this PR a board may now define _only_ a subset
   the leds and map them at the board level to the
   color LED it wants to use.

       Logical usage             Legacy default
      ------------------------+-------------
       BOARD_OVERLOAD_LED     | LED_RED
    when BOARD_HAS_CONTROL_STATUS_LEDS is defined
       BOARD_ARMED_LED        | LED_BLUE
       BOARD_ARMED_STATE_LED  | LED_GREEN

   If any of the BOARD_{OVERLOAD|ARMED|ARMED_STATE}_LED are not defined. The code output generates a null action for that  LED.

### CODE REVIEW

**Comment by davids5** [comment]
@PX4/testflights - Please test this against master.

Note only the onboard LED behavior. AMBER or RED, BLUE and GREEN leds. master and this PR should look the same on all HW except for 

FMUV5 (and FMUv5 mini RC5) - the Power (Green LED) should be on solid and the ACT (Blue LED) should blink.

Note on my FMUv4pro the leds (in the case) are installed backwards and do not work. FYI @klopezal

**Comment by santiago3dr** [comment]
ran this against master on a pixhawk 1 running v2 and a pixhawk mini with v3
blue green and red all look the same against master, not sure how to test amber

**Comment by Avysuarez** [comment]
tested with pixhawk 2.1 and 4.  is the same at the master

**Comment by davids5** [comment]
> ran this against master on a pixhawk 1 running v2 and a pixhawk mini with v3
> blue green and red all look the same against master, not sure how to test amber

@santiago3dr we would need to create a heavy load on cpu.

Here is one way to do it. (tested on a FMUv5 mini)
use `fmu mode_pwm3cap1` and then run `top` place a signal generator on FMU_CH4 with a square wave of 400,000 Hz. 3.3Vp-p. The load should go to ~93% The load light (red) will light.

**Comment by dannyfpv** [comment]
tested with pixhawk pro v4 and pixhawk mini v3. is the same at the master

**Comment by davids5** [comment]
@dagar - I would say good to go

File: src/drivers/boards/crazyflie/board_config.h [hunk 1]
```diff
@@ -71,6 +71,10 @@
 #define GPIO_LED_BLUE_L		(GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_50MHz|GPIO_OUTPUT_SET|GPIO_PORTD|GPIO_PIN2)
 
 #define BOARD_HAS_CONTROL_STATUS_LEDS	1
+#define BOARD_HAS_CONTROL_STATUS_LEDS 1
```
Comment (dagar) [inline src/drivers/boards/crazyflie/board_config.h]: duplicate line

**Comment by davids5** [comment]
@dagar - can we get this into master and on the BP for the mini?

**Pull request merged**